packages/widget.api/src/manifest.ts

```diff
@@ -49,7 +50,9 @@ export interface WidgetSetManifestContentV1 {
   inputSpec?: WidgetSetInputSpecV1;
 }
 
-export interface WidgetManifestConfigV1 {
+export type WidgetManifestConfigType = "workshopWidgetV1" | "slateWidgetV1";
```

Comment:
I'm sure you're tracking the next step to also implement the manifest parsing on publish?